app/models/venue.rb

```diff
@@ -0,0 +1,17 @@
+require 'uri'
+
+class Venue < ApplicationRecord
+
+  validates :name, presence: true
+
+  validates :url, format: { with: URI.regexp }, if: 'url.present?'
+
+  validates :address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+
+  validates :zip, presence: true
+  validates :zip, format: { with: /[\d]/,
+    message: "is not a number" }
```

Comment:
Fix this! Either make it all on one line, or organize it like you would CSS along multiple lines